lib/perl/Genome/Model/Tools/BedTools.pm

```diff
@@ -49,6 +49,19 @@ my %BEDTOOLS_VERSIONS = (
     '2.3.2' => Genome::Config::get('sw') . '/bedtools/BEDTools-2.3.2',
 );
 
+sub available_bedtools_versions {
```

Comment:
@acoffman @brummett Perl::Version and Sort::Versions are installed for 5.8.7, but not 5.10.1. version does not correctly sort the maintenance release number:
```
$ sort { $b cmp $a } map { version->parse($_)->normal } (qw/ v2.2.1 2.2.2 v2.2.3 2.2.11 2.3.0 2.1.0 /)
v2.3.0
v2.2.3
v2.2.2
v2.2.11
v2.2.1
v2.1.0
```